src/EditorFeatures/CSharp/EmbeddedLanguages/CSharpEmbeddedLanguageEditorFeaturesProvider.cs

```diff
@@ -1,28 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Composition;
-using Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages.LanguageServices;
-using Microsoft.CodeAnalysis.CSharp.Features.EmbeddedLanguages;
-using Microsoft.CodeAnalysis.Editor.EmbeddedLanguages;
-using Microsoft.CodeAnalysis.EmbeddedLanguages;
-using Microsoft.CodeAnalysis.Host.Mef;
-
-namespace Microsoft.CodeAnalysis.CSharp.Editor.EmbeddedLanguages
-{
-    [ExportLanguageService(typeof(IEmbeddedLanguagesProvider), LanguageNames.CSharp, ServiceLayer.Editor), Shared]
-    internal class CSharpEmbeddedLanguageEditorFeaturesProvider : AbstractEmbeddedLanguageEditorFeaturesProvider
```

Comment:
we used to have a mechanim by which we packages up all embedded language services together for a particular language.  That system is going away and is slowly being replaced with one where individual features (like calssification or brace matching) can pull in embedded lang helpers to augment their experiences instead.